ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/gui/PlayGamePanel.java

```diff
@@ -229,6 +231,16 @@ private JMenu createMenuBar() {
         return mOptionsMenu;
     }
 
+    private void resizeElements(int width, int height){
+        for (BoardPanel board : mGameBoards) {
```

Comment:
You can do this in one line (and more efficiently) with:
`Stream.of(mGameBoards).forEach(board -> board.rescaleBoard(width, height));`